dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/BlankApp/app.manifest

```diff
@@ -2,6 +2,15 @@
 <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
   <assemblyIdentity version="1.0.0.0" name="$projectname$.app"/>
 
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
+    <application>
+      <!--The ID below indicates app support for Windows 8
+
+      Also, it's needed for some APIs in unpackaged scenarios -->
```

Comment:
This updates the comment so that it doesn't imply WinAppSDK supports down to Win8. Is there an API example we can give here?

(Same comments for the other files)

```suggestion
      <!--The ID below indicates that the app supports features that were introduced in Windows 8
      For example this is necessary for the ... API
      For more info see
      https://docs.microsoft.com/windows/win32/sysinfo/targeting-your-application-at-windows-8-1 -->
```

Reply:
Alternative idea for the comment text:
> Inform the system that this application is compatible with OS features first introduced in Windows 8, e.g. <some API>.